apps/web/pages/v2/apps/installed/[category].tsx

```diff
@@ -96,19 +96,18 @@ interface IntegrationsListProps {
   data: inferQueryOutput<"viewer.integrations">;
 }
 
-const IntegrationsList = ({ data, variant }: IntegrationsListProps) => {
-  const separateItems = variant === "automation";
+const IntegrationsList = ({ data }: IntegrationsListProps) => {
   return (
-    <List className={classNames(separateItems ? "flex flex-col gap-6" : "")} noBorderTreatment>
+    <List className="flex flex-col gap-6" noBorderTreatment>
       {data.items.map((item) => (
         <IntegrationListItem
           name={item.name}
           slug={item.slug}
-          key={item.name}
+          key={item.title}
           title={item.title}
           logo={item.logo}
           description={item.description}
-          separate={separateItems}
+          separate={true}
```

Comment:
Always show in separate mode as AppSettings are supported by all apps now.

Reply:
I think the separate prop is not needed anymore now. So we should remove it

Reply:
I still see one usage of IntegrationListItem where separate property isn't passed to be true.So, it can't be removed yet.